### PR TITLE
More frontend tweaks

### LINKED ISSUE #75: Write further description for what domains vs. pages mean from Top 20 section
(based on user feedback)

### PR DESCRIPTION
**Pull request by leahbannon**
* Tweak and improve some of the explanation text (typos, etc.)
* Add headers and more description to Top 20 pages vs. domains, as well as adding explanation to About page to address #75 (This doesn't look perfect, and additional text will inherently look less good, but I think--in theory--the additions are important based on UX testing. I welcome design tweaks, though.)

Fixes #75.

### CODE REVIEW

**Comment by leahbannon** [comment]
pages vs. domains

screen shot 2015-01-18 at 5 12 36 pm
screen shot 2015-01-18 at 5 12 28 pm

File: demo/index.html [hunk 1]
```diff
@@ -163,13 +166,16 @@ <h2 id="current_visitors" class="data">...</h2>
...
         </p>
 
         <p>
-          Not every government website is represented in this data. Currently, the Digital Analytics Program collects web traffic from almost 300 executive branch government domains out of <a href="https://catalog.data.gov/dataset/gov-domains-api">around 1,350 domains</a> total. This data is not a representative sample of all web traffic to government websites because it only includes agencies that are participating in the program.
+          Not every government website is represented in this data. Currently, the Digital Analytics Program collects web traffic from almost 300 executive branch government domains out of <a href="https://catalog.data.gov/dataset/gov-domains-api">about 1,350 domains</a> total. This data is not a representative sample of all web traffic to government domains.
+        </p>
+
+        <p><strong>Top 20 data:</strong> "Now" data includes traffic to a specific, single page, whereas "7 Days" and "30 Days" data includes traffic to a domain AND all sub-pages within that domain. This is how the Google Analytics API releases the data.
```
Comment (konklone) [inline demo/index.html]: > This is how the Google Analytics API releases the data.

Personally, I think the section is equally descriptive without this. I'm also concerned about this baiting developers to go nitpick us on what the API docs say.

File: demo/index.html [hunk 2]
```diff
@@ -177,7 +183,7 @@ <h2 id="current_visitors" class="data">...</h2>
         </p>
 
         <p>
-          We plan to expand and document the data made available here. If you have any suggestions, or spot any issues or bugs, please <a href="https://github.com/GSA/analytics.usa.gov/issues">open an issue on GitHub</a>.
+          We plan to expand and document the data made available here. If you have any suggestions, or spot any issues or bugs, please <a href="https://github.com/GSA/analytics.usa.gov/issues">open an issue on GitHub</a> or contact the <a href="http://www.digitalgov.gov/services/dap/">Digital Analytics Program</a>.
```
Comment (konklone) [inline demo/index.html]: How about a `mailto:` for `dap@gsa.gov`?

**Comment by leahbannon** [comment]
@konklone I was thinking that "This is how Google releases the data" would help explain why we chose to display the data in this way. If that's not accurate, I think we should still explain why we chose to display the data this way -> since it is more confusing, there must be a outweighing benefit worth mentioning, right?

**Comment by konklone** [comment]
Yeah, it's more of a choice than a technical limitation. The one technical limitation we have from Google is that the Real Time API doesn't give you domains, only page titles/URLs. The "Reporting API" (which the 7/30 day tabs draw from) can give you either domains or individual pages.

To me, the outweighing benefit comes down to what's useful for that particular kind of metric. When there's a spike of live traffic to something, knowing the individual page is very relevant - just "whitehouse.gov" or "ftc.gov" wouldn't explain why that page is suddenly so popular.

It's also helpful to understand which government websites _generally_ get the most traffic, which you can only get when measuring long term traffic (the design originally included a 90-day span too). A 7-day span is sort of in-between, and both metrics would be interesting.

So to me the difference is conceptual. It's "what in the government is getting attention right now?" vs "what parts of the government see the heaviest use?", more than the specific metrics we use to arrive at those.

**Pull request merged**